ToDoApp/containers/home/Home.js

```diff
@@ -34,32 +35,77 @@ var data = [
         'second-description': 'Second Description',
         'status': false
     }
-];
-
+];*/
 class Home extends Component {
+  constructor(props){
+      super(props);
+      this.state = {
+          data:[{
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        },
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        },
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        },
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': true
+        }, 
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        }]
+    }
+    StatusBar.setBackgroundColor('#1f86ff');
+    this._handleAddData = this._handleAddData.bind(this)
+  }
+
   static navigationOptions = ({navigation}) => ({
     title: 'Home',
     headerLeft: (<View />),
     headerRight: (
         <View style={styles.headerRightContainer}>
-            <TouchableOpacity onPress={() => navigation.push('NewTask')}>
+            <TouchableOpacity onPress={() => navigation.navigate('NewTask',{handleAddData: navigation.getParam('addData')})}>
                 <Text style = {styles.headerRightText}>+</Text>
+
             </TouchableOpacity>
         </View>
       ),
   });
 
   onPressFuction = (item) => { this.props.navigation.push('Details', {...item}) }
   
+  componentDidMount(){
+    this.props.navigation.setParams({addData: this._handleAddData});
+  }
+
+  _handleAddData(taskTitle,taskDescription){
```

Comment:
you should use arrow functions so to avoid having to bind `this`
`_handleAddData = (taskTitle,taskDescription) => {`